stock-tracker/src/features/keyStats/component/keyStats.tsx

```diff
@@ -5,15 +5,19 @@ import ErrorMessage from "../../error/error";
 import "./KeyStats.css";
 import { AppState } from "../../../store/rootReducer";
 
+const numberWithCommas = (x: number) => {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
q: That's one nice piece of Regex! You could also use the inbuilt JavaScript method to format numbers? `Intl.NumberFormat()`

Reply:
I saw your implementation of `Intl.NumberFormat() `which I thought was neat! Was just using Regex for this to learn how Regex works.